pyls_spyder/plugin.py

```diff
@@ -81,7 +103,7 @@ def pyls_document_symbols(config: Config,
                 cell_name = 'Unnamed cell {0}'.format(unnamed_cell)
                 unnamed_cell += 1
 
-            if not group_cells:
+            if not group_cells or cell_rule != CELL_PERCENTAGE:
```

Comment:
Why did you add `or cell_rule != CELL_PERCENTAGE` here?

Reply:
Because nesting is not well defined for the other kind of cells

Reply:
Right, good point.